docs/docsite/rst/getting_started_ee/run_community_ee_image.rst

```diff
@@ -0,0 +1,26 @@
+.. _running_community_ee_image:
+
+Running Ansible with the community EE image
+===========================================
+
+You can run ansible without the need to build a custom EE. 
+Use the ``community-ee`` image that includes ``ansible-core`` and a set of Ansible community collections.
```

Comment:
Looking at https://github.com/ansible-community/ansible-build-data/pull/278/files, I don't see any community collections. It's just `ansible.*` ones.